mailchimp-sdk-main/src/main/java/com/mailchimp/sdk/main/Mailchimp.kt

```diff
@@ -69,10 +69,9 @@ open class Mailchimp protected constructor(
     }
 
     protected fun setupSdk(debugModeEnabled: Boolean) {
-        if (debugModeEnabled) {
+        if (debugModeEnabled && Timber.treeCount() == 0) {
```

Comment:
I think Timber handles this now.

Reply:
Can you confirm? I am pretty sure without this consumers of this SDK would end up with two log statements for every log (if they are using `Timber`). You can test with our app if you want.